src/containers/NotablePerson/NotablePerson.jsx

```diff
@@ -14,6 +14,7 @@ import { Footnotes } from '../../components/Footnotes/Footnotes';
 import { Section } from '../../components/Section/Section';
 import { Separator } from '../../components/Separator/Separator';
 import * as eventTypes from '../../constants/eventTypes';
+import SpeedDails from '../../components/SpeedDail/SpeedDail';
```

Comment:
This is coming up as `undefined`. The name that's exported from `SpeedDail/SpeedDail` is `SpeedDials`, so it would have to be imported as `{ SpeedDials }`.

I ran this branch locally and fixed it in order to be able to render the page.